web/themes/custom/digital_gov/digital_gov.theme

```diff
@@ -43,17 +43,17 @@ function digital_gov_preprocess_node(array &$variables): void {
       $variables['show_archive_warning'] = $created->getPhpDateTime() <= $before_date->getPhpDateTime();
       break;
 
-    case 'event':
-      $now = new DrupalDateTime();
-      $startDate = $variables['node']->field_start_date->date;
-      $variables['start_date'] = $startDate->format('l, F j, Y', ['timezone' => date_default_timezone_get()]);
-      $variables['start_time'] = $startDate->format('g:i A', ['timezone' => date_default_timezone_get()]);
-      $variables['is_future_event'] = $startDate->getPhpDateTime() > $now->getPhpDateTime();
-
-      if ($variables["node"]->field_end_date->value) {
-        $endDate = $variables["node"]->field_end_date->date;
-        $variables['end_time'] = $endDate->format('g:i A T', ['timezone' => date_default_timezone_get()]);
-      }
+    // case 'event':
```

Comment:
can you take a look at this... when it is uncommented i get an error about the date set up for events but when it commented out its working so wasn't for sure if this was needed or maybe needs to be udpated?